frontend/micro-ui/web/micro-ui-internals/packages/react-components/src/atoms/UploadFile.js

```
@@ -189,59 +185,93 @@ const UploadFile = (props) => {
+              backgroundColor: "#7a2829",
+            }}
             textStyles={styles}
             type={props.buttonType}
           />
-            {props?.uploadedFiles?.map((file, index) => {
-              const fileDetailsData = file[1]
-              return <div className="tag-container" style={extraStyles ? extraStyles?.tagContainerStyles : null}>
+          {props?.uploadedFiles?.map((file, index) => {
+            const fileDetailsData = file[1];
+            return (
+              <div className="tag-container" style={extraStyles ? extraStyles?.tagContainerStyles : null}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)